guides/12-general/9-routing-new-markets.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_label: "Routing for New Markets"
+description: Learn more about how to get your markets/pools routed on Jupiter.
+slug: "/routing-new-markets"
+title: Routing for New Markets
```

Comment:
Not only for new markets, it's all markets, consider "Market Routing Requirements" or something similar?